lib/chef/provisioning/aws_driver/aws_taggable.rb

```diff
@@ -0,0 +1,18 @@
+module Chef::Provisioning::AWSDriver
+# This module is meant to be included in a resource that is taggable
+# This will add the appropriate attribute that can be converged by the provider
+# TODO it would be nice to not have two seperate modules (taggable/tagger)
```

Comment:
IMO one of the confusing things about provisioning is the way it incompletely tries to blur the resource/provider distinction, so it seems fine to keep them separate and reduce the magic. If there's a way to somehow give clear feedback to the user when something is wrong--like they pass `aws_tags` into `machine_options` when the resource or provider isn't set up for tagging--then we keep to the way Chef resources are generally presented, and minimize the spooky-action-at-a-distance.